### PR TITLE
Feature/login error handling 1

### PR DESCRIPTION
**Pull request by MTCawthray**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- This branch creates basic error handling in the login components.
- Now, when an input field is left empty, the submit buttons are disabled.
- If a user is creating an account and their passwords do not match, they are greeted with text informing them that their passwords did not match.

## What (if anything) did you refactor?
- NA

## Were there any issues that arose?
- The new errors being returned by the BE can cause the react app to shut down. The purpose of these front end form error handling is to mitigate that situation.
- Currently, for an existing user creating an account, if the username or password do not match what is on file, the BE response breaks the app. We need to have BE revert back to the success true/false response in order to show our error messaging and not break the app.

## Is there anything that you need from your teammates?
- Please see above!

## Any other comments, questions, or concerns?
- NA

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor

## How Has This Been Tested?
- [x] open index.html
- [ ] console.log()
- [x] dev tools
- [ ] Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -46,8 +47,8 @@ const Login = ({assignUser, user, updateTypes}) => {
         }
       })
     } else {
-      alert('your password did not match')
-      //do some sort of error handling
+      setMessage('your password did not match')
+      setTimeout(() => {setMessage('')}, 4000)
```
Comment (hayleyw7) [inline src/components/Login/Login.js:49]: I love that it goes away. Cool styling idea.

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -94,9 +95,13 @@ const Login = ({assignUser, user, updateTypes}) => {
               type="submit"
               className="submit-btn next"
               onClick={createAccount}
+              disabled={!username || !newPassword || !passCheck }
             >
               Create Account
             </button>
+
+            { message && <p className="message" >{message}</p>}
```
Comment (hayleyw7) [inline src/components/Login/Login.js:102]: Great conditional here.

**Review by hayleyw7 — APPROVED** [review]
Thanks for this quick fix. Can't wait to see it in action.

File: src/components/Login/Login.js (hunk 1 above)
Comment (hayleyw7) [inline src/components/Login/Login.js:49]: Looks great!

**Review by hayleyw7 — APPROVED** [review]

File: src/components/Login/Login.js [hunk 3]
```diff
@@ -46,8 +47,8 @@ const Login = ({assignUser, user, updateTypes}) => {
         }
       })
     } else {
-      alert('your password did not match')
-      //do some sort of error handling
+      setMessage('your password did not match')
```
Comment (nataliemcintyre2021) [inline src/components/Login/Login.js:48]: Nice use of state here for handling error message.

**Pull request closed without merging**